src/client/src/pages/overview/sidePanelContent/importer/importModal.tsx

```diff
@@ -26,42 +26,52 @@ const ImportModal = ({
 }: ImportModalProps) => {
   const { showAlert } = useContext(AlertContext);
   const { t } = useTranslation();
+  const [fileType, setFileType] = useState<string | null>(null); // Track file type
```

Comment:
PP:
```suggestion
  const [fileType, setFileType] = useState<string>("");
```
und den null check ggf angleichen